Makefile

```diff
@@ -471,8 +438,14 @@ pr:
 
 .PHONY: golangci-lint
 golangci-lint:
-	@hash golangci-lint > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
-		export BINARY="golangci-lint"; \
-		curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $(GOPATH)/bin v1.17.1; \
-	fi
 	golangci-lint run --deadline=3m
+
+prepare-tools:
+		GO111MODULE="on" $(GO) get -u github.com/go-swagger/go-swagger/cmd/swagger@v0.20.1
+		$(GO) get -u github.com/kisielk/errcheck
```

Comment:
Can't we activate GO111MODULE="on" also for the others tools?

Reply:
Done.

Reply:
I think that is okay.